bundles/framework/rpc/instance.js

```diff
@@ -499,17 +237,17 @@ Oskari.clazz.define(
          *
          */
         stop: function () {
-            var me = this,
-                sandbox = this.sandbox,
-                p;
+            const me = this;
+            const sandbox = this.sandbox;
+            let p;
 
             for (p in me.eventHandlers) {
-                if (me.eventHandlers.hasOwnProperty(p)) {
+                if (Object.prototype.hasOwnProperty.call(me.eventHandlers, p)) {
                     sandbox.unregisterFromEventByName(me, p);
                 }
             }
             for (p in me.requestHandlers) {
-                if (me.requestHandlers.hasOwnProperty(p)) {
+                if (Object.prototype.hasOwnProperty.call(me.requestHandlers, p)) {
                     sandbox.removeRequestHandler(p, this);
                 }
             }
```

Comment:
Maybe just do:
```
Object.keys(me.eventHandlers).forEach(eventName => sandbox.unregisterFromEventByName(me, eventName);
Object.keys(me.requestHandlers).forEach(requestName => sandbox.removeRequestHandler(requestName, me);
```
Funky that the instance is the first param for events but second for requests :) Haven't noticed that before.